com.adamcake.Bolt.yml

```diff
@@ -0,0 +1,57 @@
+app-id: com.adamcake.Bolt
+default-branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: bolt
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --env=PATH=/app/bin:/usr/bin:/app/jre/bin
+modules:
+  - name: python-pillow
+    buildsystem: simple
+    build-commands:
+      - pip3 install . --prefix=${FLATPAK_DEST}
+    build-options:
+      build-args:
+        - --share=network
+    sources:
+      - type: archive
+        url: https://github.com/python-pillow/Pillow/archive/refs/tags/10.0.0.tar.gz
+        sha256: 535d17e830427bec163027114ded1def9ab0350c99bf1d8cb10535032967f3a5
+  - name: bolt
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBOLT_BINDIR=bin
+    sources:
+      - type: archive
+        dest: cef/dist
+        url: https://adamcake.com/cef/cef-114.0.5735.201-linux64-minimal-ungoogled-flatpak.tar.gz
+        sha256: b9c09ebfdf2cbef10cb43527a90d01f02a7397fd3a504bc8c05c5b78767dec47
+      - type: git
+        url: https://github.com/Adamcake/Bolt.git
+        tag: 0.1.0
+        commit: fd36ad25433fa55ab83acacac9303ee9357a8539 
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+  - shared-modules/gtk2/gtk2.json
+  - name: openssl
+    buildsystem: simple
```

Comment:
this should be autotools.

Reply:
When I tried setting this to autotools it didn't work as openssl doesn't come with a makefile and also has "no autogen, autogen.sh or bootstrap". The `./config` line generates the makefile, so it would need to be run before any autotools steps. Is there a way to convey that to flatpak-builder? I went through the command reference at the time and didn't find a way. Thanks